app/views/devise/passwords/new.html.erb

```diff
@@ -13,8 +13,7 @@
                 <%= f.label :email %><br />
                 <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
             </div>
-
-            <%= f.submit "Send me reset password instructions", class: "ui button primary" %>
+            <%= f.submit t('.reset_button'), class: "ui button primary" %>
```

Comment:
This name is super! It clearly expands the meaning of the translation key. It allows to change translations (say from `Reset` to `Send me reset password instructions` or any other) without changing the code. Also, opening translation file it's immediately clear how the key is used in UI (`button`, in this case).